app/controllers/PropertyRepresentationController.scala

```diff
@@ -68,4 +71,76 @@ class PropertyRepresentationController @Inject() (
      }
    }
 
+  def appointableToAgent(ownerId: Long,
+                         agentCode: Long,
+                         checkPermission: Option[String],
+                         challengePermission: Option[String],
+                         paginationParams: PaginationParams,
+                         sortfield: Option[String],
+                         sortorder: Option[String],
+                         address: Option[String],
+                         agent: Option[String]) = authenticated { implicit request =>
+
+    groupAccountsConnector.withAgentCode(agentCode.toString) flatMap {
+      case Some(agentGroup) => propertyLinksConnector.appointableToAgent(
+        ownerId = ownerId,
+        agentId = agentGroup.id,
+        checkPermission = checkPermission,
+        challengePermission = challengePermission,
+        params = paginationParams,
+        sortfield = sortfield,
+        sortorder = sortorder,
+        address = address,
+        agent = agent).map(x => Ok(Json.toJson(x)))
+      case None =>
+        Logger.error(s"Agent details lookup failed for agentCode: $agentCode")
+        NotFound
+    }
+  }
+
+
+  /** *
+    * Make two calls to the Search/Sort API
+    * the first call returns the results based on supplied filters and sortfield
+    * the second call is used only to allow us to get the count of PENDING representation requests
+    */
+  def forAgentSearchAndSort(organisationId: Long,
```

Comment:
what is this method to support?

Reply:
I think we agreed to move this endpoint from PropertyLinkingController to the PropertyRepresentationController. Gets call in Dashboard YourPropertiesController.show()
and PLF RepresentationController.listRepresentationRequest()